install_config/install/disconnected_install.adoc

```diff
@@ -194,16 +200,26 @@ Replace `<tag>` with `{latest-int-tag}` for the latest version.
 endif::[]
 +
 ----
+# docker pull registry.access.redhat.com/openshift3/container-engine:<tag>
+# docker pull registry.access.redhat.com/openshift3/efs-provisioner:<tag>
```

Comment:
container-engine and efs-provisioner images should belong to "the required {product-title} containerized components"